src/marks/mark.py

```diff
@@ -97,20 +101,83 @@ def cursorless_mark(m) -> str:
     return m.cursorless_line_number_simple
 
 
-def on_ready():
-    init_csv_and_watch_changes(
-        "special_marks",
+DEFAULT_COLOR_ENABLEMENT = {
```

Comment:
Yeah I know not great to hardcode this stuff.  But we won't need to once we do it using the new state sharing stuff so I think this is ok for now

Reply:
@AndreasArvidsson Ok so one problem with this hard coding is that if we turn some shapes on by default extension side, we would have to update this code and then there would be mismatch until they update talon side

We could maybe have cursorless extension write default enablement to json in post-install script?